ols-2.md

```diff
@@ -51,6 +51,7 @@ photos:
 {% assign p-hosts = all-hosts | remove_first: ', ' | split: ", " | uniq | sort %}
 
 # The OLS-2 program
+
```

Comment:
Could you please remove the extra line here? It will otherwise break the template

Reply:
It has been removed